src/plasma/plasma_manager.h

```diff
@@ -59,7 +59,7 @@ void PlasmaManagerState_free(PlasmaManagerState *state);
  * manager.
  */
 void process_transfer(event_loop *loop,
-                      ObjectID object_id,
+                      ray::ObjectID object_id,
```

Comment:
not super critical, but all of the `ray::ObjectID object_id` in this file should be `const ray::ObjectID &object_id`.

Reply:
Yeah, let's do this incrementally!